modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/ClusterManagementGroupManager.java

```diff
@@ -526,17 +526,10 @@ private TopologyEventHandler cmgLeaderTopologyEventHandler(CmgRaftService raftSe
         return new TopologyEventHandler() {
             @Override
             public void onAppeared(ClusterNode member) {
-                raftService.readClusterState()
-                        .thenAccept(state -> {
-                            if (state != null) {
-                                sendClusterState(state, member)
-                                        .whenComplete((v, e) -> {
-                                            if (e != null) {
```

Comment:
This has been dropped. Should it be useful for debugging problems (maybe even as a WARNING)?

Reply:
`sendClusterState()` might throw something that is not covered by the code on line 593 (like an exception when creating a message). This is currently highly unlikely, but saw such cases in other projects (when due to a stupid coding error an NPE is thrown, for instance). If such an un unforseen exception is thrown, we will still not see it, and it will be pretty difficult to figure out what's going on.

IMO, better safe than sorry :)

Reply:
But otherwise we will log the same error twice, I don't like it